mkdocs/docs/metrics/defining-iter8-metrics.md

```diff
@@ -0,0 +1,640 @@
+        requestCount: myns/request-count
+        objectives:
+        - metric: myns/name-count
+          lowerLimit: 50
+        - metric: myns/cpu-utilization
+          upperLimit: 90
+        - metric: myns/average-sales
+          lowerLimit: "250.0"
+      duration:
+        intervalSeconds: 10
+        iterationsPerLoop: 10
+      versionInfo:
+        baseline:
+          name: current
+          variables:
+          - name: revision
+            value: sample-app-v1 
+          - name: userfilter
+            value: 'usergroup!~"wakanda"'
+        candidates:
+        - name: candidate
+          variables:
+          - name: revision
+            value: sample-app-v2
+          - name: userfilter
+            value: 'usergroup=~"wakanda"'
+    ```
+
+For the sample experiment above, Iter8 will use two HTTP(S) queries to fetch metric values, one for the baseline version, and another for the candidate version.
+
+=== "Prometheus"
+
+    For the baseline version, Iter8 will send an HTTP(S) request with a single parameter named `query` whose value equals:
+    ```
+    sum(increase(revision_app_request_latencies_count{service_name='current',usergroup!~"wakanda"}[600s])) or on() vector(0)
+    ```
+
+=== "New Relic"
+    For the baseline version, Iter8 will send an HTTP(S) request with a single parameter named `nrql` whose value equals:
+    ```
+    SELECT count(appName) FROM PageView WHERE revisionName='sample-app-v1' SINCE 600 seconds ago
+    ```
+    
+=== "Sysdig"
+    For the baseline version, Iter8 will send an HTTP(S) request with the following JSON body:
+    ```json linenums="1"
+    {
+      "last": 600,
+      "sampling": 600,
+      "filter": "kubernetes.app.revision.name = 'sample-app-v1'",
+      "metrics": [
+        {
+          "id": "cpu.cores.used",
+          "aggregations": { "time": "avg", "group": "sum" }
+        }
+      ],
+      "dataSourceType": "container",
+      "paging": {
+        "from": 0,
+        "to": 99
+      }
+    }
+    ```
+
+=== "Elastic"
+    For the baseline version, Iter8 will send an HTTP(S) request with the following JSON body:
+    ```json linenums="1"
+    {
+      "aggs": {
+        "range": {
+          "date_range": {
+            "field": "date",
+            "ranges": [
+              { "from": "now-600s/s" } 
+            ]
+          }
+        },
+        "items_to_sell": {
+          "filter": { "term": { "version": "sample-app-v1" } },
+          "aggs": {
+            "avg_sales": { "avg": { "field": "sale_price" } }
+          }
+        }
+      }
+    }
+    ```
+
+The placeholder `$elapsedTime` has been substituted with 600, which is the time elapsed since the start of the experiment. The other placeholders have been substituted based on information associated with the baseline version in the experiment. Iter8 builds and sends an HTTP request in a similar manner for the candidate version as well.
```

Comment:
Can we be more specific here? Perhaps:

The other placeholders have been substituted based on the values defined in the variables portion of the baseline _versionInfo_ in the experiment.

Reply:
Done.